morse_code.rb

```diff
@@ -0,0 +1,22 @@
+def morse_code(code)
+  codes = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O', '.--.' => 'P',
+    '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U', '...-' => 'V',
+    '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z'
+  }
+  codes[code]
+end
```

Comment:
- [SUGGESTION] Your methods work fine. I wish their naming could be given some kind of meaning. I mean, in relation to what they are supposed to do. For example, this could be `decode_char`. Why? Its primary purpose is to decode for us a single char. 

  Naming methods properly is key to code maintainability, especially in a collaborative environment. The name of a method helps a developer figure out what it's supposed to do even without commenting on that method.